tests/src/test/java/org/corfudb/test/docker/infrastructure/HelloWorldTest.java

```diff
@@ -0,0 +1,27 @@
+package org.corfudb.test.docker.infrastructure;
+
+import lombok.extern.slf4j.Slf4j;
+import org.corfudb.test.AbstractCorfuUniverseTest;
+import org.corfudb.test.TestGroups;
+import org.corfudb.universe.infrastructure.docker.universe.group.cluster.DockerCorfuCluster;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+@Slf4j
+@Tag(TestGroups.VERSION_COMPATIBILITY)
+public class HelloWorldTest extends AbstractCorfuUniverseTest {
```

Comment:
Could you please rename it to something more meaningful

Reply:
Renamed, thanks!